### ISSUE TITLE
Make it possible to hide white side panel line when side panel is collapsed

### ISSUE THREAD

**Issue by isak102** [linuxmint/xed#626]
```
 * Xed version (xed --version): 3.4.5
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...): Arch
```
Would it be possible to hide this line when the side panel is collapsed? This is the right edge of the side panel but it is still visible, even when collapsed.
![2024-Jan-11Thu_21 56 37](https://github.com/linuxmint/xed/assets/90389894/3c631cbb-55c9-4662-8ef6-7f30384ad9fe)

**Comment by J0sephDavis** [comment]
## TLDR:

To fix your problem change themes in the Xed preferences OR
edit your theme at /usr/share/gtksourceview-4/styles/classic.xml, line:45 #eeeeec should be #404040

## Why is the white bar there?

The problem is that the classic theme's TextView border has the wrong background color for the theme.
If you go to preferences->editor and enable Display->Display Line Numbers, the region will expand and be filled with line numbers.
The solution is to go to preferences->theme and change to one of the themes that  have a good looking text view border.
![Image](https://github.com/user-attachments/assets/1fb8b917-4888-4d58-b41e-fedf82f65a14)

## Looks fine in GEdit

It looks like both  Tango and Classic have the same issue of using a white background color, with white text.
From an online guide I spied the GEdit theme page and it seems to use the proper dark theme colors:
![Image](https://itsfoss.com/content/images/size/w1000/2023/07/Fonts-and-Colors-tab-change-default-themes.png)
Not sure whats different about our gtk installation vs theirs, but we can just change it ourselves.

## How to change the default style's background color

Go to: `/usr/share/gtksourceview-4/styles`
edit the xml file for the ui style you care about. e.g.,`classic.xml`
Then, change line:45 <style name="line-numbers" background="#eeeeec">
You want the background color to be something dark, such as `#404040` which, after using the inspector, I have determined is the background color of the textview in my current instance of Xed. After restarting Xed, you can now see the classic theme is readable, and the dumb white bar is gone:

![Image](https://github.com/user-attachments/assets/54dcf9af-885a-4089-8168-b032ae3fa3f8)

## Can't find the styles folder?

Remember, edit the gtksourceview-4/styles, not gtksourceview-3/styles. If you can't find your styles folder / it isn't where I said it was, you can try some of these folders, which I got from the [gnome wiki](https://wiki.gnome.org/Projects(2f)GtkSourceView(2f)StyleSchemes.html)
    ~/.local/share/gtksourceview-4/styles/
    /usr/share/gnome/gtksourceview-4/styles/
    /usr/local/share/gtksourceview-4/styles/
    /usr/share/gtksourceview-4/styles/

**Comment by isak102** [comment]
Thanks!
